R/pkg/inst/worker/worker.R

```diff
@@ -27,6 +27,58 @@ elapsedSecs <- function() {
   proc.time()[3]
 }
 
+computeHelper <- function(mode, partition, serializer, deserializer, key,
```

Comment:
Hi @sun-rui , this is good, however time accumulation might lead to very large numbers and numerical type limitations.
I was thinking to accumulate the diffs such as:
inputComputeDiff += compute - input and computeOutputDiff += output - compute.

What do you think ?